var/spack/repos/builtin/packages/py-jdatetime/package.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyJdatetime(PythonPackage):
+    """jdatetime is Jalali implementation of Python’s datetime module"""
```

Comment:
This is a long shot, but maybe the smart quote in this docstring is what is causing the problem? That would explain why only Python 2 is affected and not Python 3. Try changing this to a regular quote.
```suggestion
    """jdatetime is Jalali implementation of Python's datetime module"""
```